### DISCUSSION TITLE
3/16 daily spring

### DISCUSSION THREAD

**Discussion by revature-ives** [algoives/TestBankApp#31] [Daily Scrum]
Hello @RahmmaAwali @ClarkeWilliford  welcome to daily spring

**Comment by ClarkeWilliford** [comment]
yesterday we worked on designing the overall views. Today I'm going to work on actually building the login screens, starting with learning more about SQlite3

**Comment by RahmmaAwali** [comment]
Yesterday we create github and design the UI for all view controller . Today I will do the user interface (uI).

**Comment by revature-ives** [comment]
**What I did yesterday**

Yesterday I work defining the data for the database, entities

**Obstaclues**

I need to understand better relationship many to many in sqlite

**What I will do today:**

I going work in the many to many relationships on the database
